packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -957,9 +957,12 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
     });
   }
 
-  handlePanelTabClick(): void {
+  handlePanelTabClick(e: MouseEvent): void {
```

Comment:
Random side note - could use `Panel.onTabClicked` prop instead of adding the event listener to glEventHub directly from here, but no big deal

Reply:
I'll make that change. Seems like a good cleanup and IMO the fewer places we listen for golden-layout events the better